### DISCUSSION TITLE
What is your approach for nested task?

### DISCUSSION THREAD

**Discussion by azzamsa** [ransome1/sleek#656] [Q&A]
Hi, folk! :wave: 

I have a question about handling nested tasks:

1)

```
- Going to City
- Going to City - Buy groceries
- Going to City - Buy milk
```

2)

```
- Going to City - Buy groceries +Travel @City
- Going to City - Buy milk  +Travel @City
```

Which one is your preferred approach? (1) or (2)?
Or do you have better approaches?

Thank you! :icecream:

**Comment by ransome1** [comment]
What is the question? :)

**Reply by azzamsa** [reply]
Which on is your preferred approach, (1) or (2)?

Or do you have better approach?

**Comment by fredofredofred** [comment]
Assuming the project / other contexts aren't already specific to the task, I would make 'Going to City' it's own context @CityGo, that way you can filter and just see related items.  

So:
 
Buy groceries +Travel @City @CityGo
Buy milk  +Travel @City @CityGo
